eng/dependabot/Packages.props

```diff
@@ -3,6 +3,8 @@
   <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Packages.props. -->
 
   <ItemGroup>
+    <PackageVersion Include="WixToolset.Dtf.WindowsInstaller" Version="4.0.6" />
```

Comment:
Not this version either. Sorry, we're bound to an internal copy of WiX 3.14 right now. We haven't began the migration work for 5.0 and we're not jumping to 4 because of servicing policies end product lifetime cycles. When 6 releases, 4 will be going out of support much faster than 3.x ever did.